react/components/AvailableFilters.js

```diff
@@ -23,7 +23,9 @@ const Filter = ({
   initiallyCollapsed,
   navigateToFacet,
   lazyRender,
+  openFiltersMode = 'many',
```

Comment:
Just a suggestion, but how about calling this mode `multiple` instead of `many`?